partner_communication/wizards/email_template_preview_view.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Joel Vaucher <jvaucher@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import models, api, fields
+
+
+class TemplatePreview(models.TransientModel):
+    _inherit = "email_template.preview"
+
+    res_id = fields.Reference(selection=lambda s: s._reference_models(),
+        default=lambda s: s._default_model())
+
```

Comment:
res_id is a selection field in the parent, some trick has to be use to make it work